lib/DDG/Spice/Github.pm

```diff
@@ -13,13 +14,43 @@ attribution web => ['http://dylansserver.com','Dylan Lloyd'],
             email => ['dylan@dylansserver.com','Dylan Lloyd'];
 
 triggers startend => "github";
-spice to => 'https://api.github.com/legacy/repos/search/$1?callback={{callback}}';
+
+my @triggers = share("triggers.txt")->slurp;
+triggers startend => @triggers;
```

Comment:
After a little more consideration, I'm afraid that given the number of dictionary words that exist in the triggers.txt I'm not sure we should allow the triggers to also be the languages. It's likely that this will cause a lot of false-positive triggering.

Let's stick with `github` as the trigger and if a language exists in the query, use that to specify the language of the result repos.